docs/tutorials/command_line_client.md

```diff
@@ -349,16 +349,17 @@ synapse list [-h] [-r] [-l] [-m] syn123
 
 ### `config`
 
-Create or modify a Synapse configuration file.
+Create or modify a Synapse configuration file. This command interactively prompts for a username and a Synapse Personal Access Token (Auth Token) and writes them to the ~/.synapseConfig file.
+Supports multiple profiles.
 
 ```bash
-synapse config [-h]
+synapse config [-h] [--profile PROFILE_NAME]
```

Comment:
I tried: `synapse config -h --profile default` and it didn't quite work. I didn't see help message for profile flag: 
```
(venv) lpeng@MacBookPro synapsePythonClient % synapse config -h --profile default
usage: synapse config [-h]

options:
  -h, --help  show this help message and exit
```